tests/__init__.py

```
@@ -46,6 +46,23 @@
 		True
 		>>>
 
+	Testcase 1: Load tests
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)